ayon_api/server_api.py

```diff
@@ -1228,7 +1228,8 @@ def enroll_event_job(
         target_topic,
         sender,
         description=None,
-        sequential=None
+        sequential=None,
+        filter=None,
```

Comment:
Missing docstring and I do expect it should be `filters`.
```suggestion
        filters=None,
```

Also `filter` is built-in python function - would be bad name.

Reply:
Ah yeah, i just went with `filter` cause that's what's on the backend :skull: 
We probably should change it there too...
https://github.com/ynput/ayon-backend/blob/develop/api/events/enroll.py#L84